### PR TITLE
source-high-level contribution from Stockotaco

### PR DESCRIPTION
**Pull request by Stockotaco**
Thanks for your contribution to the Airbyte Marketplace, @Stockotaco!

Please complete the following items, which we aim to automate soon:
- [x] Update the list of `allowedHosts` in `metadata.yaml` to include only the list of host domains your connector should have access to. 
- [x] Delete empty lines between rows in the list of streams in the generated doc file
- [x] If you'd like to upload an icon for your connector, upload an SVG file into a comment on the PR - Your reviewer will help with sizing it appropriately. If you don't have an icon and want to keep the Airbyte icon, just check this box to indicate that.

There are two ways to address this:
1. You can go into your PR, and edit files in the web UI directly
2. You can fetch the `Stockotaco:Stockotaco/builder-contribute/source-high-level` branch and edit locally

Once you've done this, a reviewer will take a look. Reviewer, please:
- [x] Run the `/format-fix` slash command to fix any formatting issues

### CODE REVIEW

**Comment by erohmensing** [comment]
high-level

This is the SVG file pre-resizing

**Comment by nora-airbyte** [comment]
no-bg
blue-bg

Resized + cleaned up icon. We don't have icons with a full coverage background, so created two options here: a partial coverage bg and a transparent bg.

File: airbyte-integrations/connectors/source-high-level/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# High Level
+This directory contains the manifest-only connector for `source-high-level`.
+
+Proxy connector for Go High Level (Lead Connector). Requires a paid subscription to the proxy service.
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-high-level/README.md:4]:
```suggestion
Proxy connector for [Go High Level](www.gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```

File: docs/integrations/sources/high-level.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# High Level
+Proxy connector for Go High Level (Lead Connector). Requires a paid subscription to the proxy service.
```
Comment (erohmensing) [inline docs/integrations/sources/high-level.md:2]:
```suggestion
Proxy connector for [Go High Level](www.gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```

File: docs/integrations/sources/high-level.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## Changelog
+
+<details>
+  <summary>Expand to review</summary>
+
+| Version          | Date       | Subject        |
+|------------------|------------|----------------|
+| 0.0.1 | 2024-08-23 | Initial release by Stockotaco via Connector Builder|
```
Comment (erohmensing) [inline docs/integrations/sources/high-level.md:36]:
```suggestion
| 0.0.1 | 2024-08-23 | Initial release by [@Stockotaco](https://github.com/stockotaco) via Connector Builder|
```

**Review by erohmensing — APPROVED** [review]

File: airbyte-integrations/connectors/source-high-level/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# High Level
+This directory contains the manifest-only connector for `source-high-level`.
+
+Proxy connector for [Go High Level](www.gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-high-level/README.md:4]:
```suggestion
Proxy connector for [Go High Level](https://gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```

File: docs/integrations/sources/high-level.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# High Level
+Proxy connector for [Go High Level](www.gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```
Comment (erohmensing) [inline docs/integrations/sources/high-level.md:2]:
```suggestion
Proxy connector for [Go High Level](https://gohighlevel.com) (Lead Connector). Requires a paid subscription to the proxy service.
```

**Review by erohmensing — APPROVED** [review]

**Pull request merged**